### ISSUE TITLE
Uncaught TypeError: re-find must match against a string.

### ISSUE THREAD

**Issue by m0rtis0** [mauricioszabo/atom-chlorine#234]
[Enter steps to reproduce:]

1. Try to connect to REPL

**Atom**: 1.54.0 x64
**Electron**: 6.1.12
**OS**: "Arch Linux"
**Thrown From**: [chlorine](https://github.com/mauricioszabo/atom-chlorine) package 0.10.2

### Stack Trace

Uncaught TypeError: re-find must match against a string.

```
At /home/m0rtis/.atom/packages/chlorine/lib/cmds.js:1029

TypeError: re-find must match against a string.
    at Xn (/packages/chlorine/lib/cmds.js:1029:103)
    at /packages/chlorine/lib/cmds.js:2852:371
    at pk.h.Ge (/packages/chlorine/lib/cmds.js:723:431)
    at vk (/packages/chlorine/lib/cmds.js:725:348)
    at Function.xk.h (/packages/chlorine/lib/cmds.js:728:206)
    at /packages/chlorine/lib/cmds.js:2832:355)
    at Socket.emit (events.js:200:13)
    at TCP.<anonymous> (net.js:586:12)
```

### Commands

```
     -4:06.0 core:undo (input.hidden-input)
     -3:59 vim-mode-plus:activate-insert-mode (input.hidden-input)
     -3:58.8.0 core:move-left (input.hidden-input)
     -3:58.3.0 core:backspace (input.hidden-input)
  2x -3:57.7.0 core:move-right (input.hidden-input)
     -3:57.1.0 core:backspace (input.hidden-input)
     -3:56.5.0 intentions:highlight (input.hidden-input)
     -3:56.4.0 core:save (input.hidden-input)
     -3:51.6.0 intentions:highlight (input.hidden-input)
     -3:48.8.0 build:trigger (input.hidden-input)
     -3:33.8.0 vim-mode-plus:activate-normal-mode (input.hidden-input)
     -3:13.7.0 build:trigger (input.hidden-input)
  3x -2:43.2.0 core:backspace (input.hidden-input)
     -0:20.8.0 chlorine:connect-socket-repl (input.hidden-input)
     -0:20.1.0 core:focus-next (input.input-text)
     -0:05.7.0 chlorine:connect-socket-repl (input.hidden-input)
```

### Non-Core Packages

```
bracket-colorizer 1.4.0 
build 0.70.0 
busy-signal 2.0.1 
chlorine 0.10.2 
ink 0.12.5 
intentions 1.1.5 
linter 3.3.0 
linter-kondo 1.2.0 
linter-ui-default 3.2.4 
parinfer 1.24.1 
vim-mode-plus 1.36.4 
which-key 0.3.3 
```

**Comment by mauricioszabo** [comment]
Seems like an autocomplete issue, will try to check what's wrong.

**Comment by mauricioszabo** [comment]
Hi, @m0rtis0. Just checked, it's not an autocomplete issue. Can you give me more information? Specially:

1. Are you stil seeing this problem? 
2. Did you have it while trying to connect to Socket REPL or nREPL?
3. Can you reproduce it?
4. Did you install Atom with pacman or by downloading some binary?

Thanks :)

**Comment by mauricioszabo** [comment]
I think it's fixed on v0.10.3. If you still see this issue, please re-open it and I'll try to debug it even more. Thanks a lot :)
